RetroBar/Languages/hrvatski.xaml

```diff
@@ -20,6 +20,15 @@
         <s:String>Desno</s:String>
         <s:String>Dno</s:String>
     </x:Array>
+    <s:String x:Key="rowcount_text">Broj _redova:</s:String>
+    <s:String x:Key="rowcount_tip">Broj redova na programskoj traci kad je na vrhu ili dnu.</s:String>
+    <x:Array x:Key="rowcount_options" Type="s:Int32">
+        <s:Int32>1</s:Int32>
+        <s:Int32>2</s:Int32>
+        <s:Int32>3</s:Int32>
+        <s:Int32>4</s:Int32>
+        <s:Int32>5</s:Int32>
+    </x:Array>
```

Comment:
Because `rowcount_options` is no different from the English version, could you leave that one out?

Reply:
Alright., done